core/src/main/java/com/radixdlt/ledger/LedgerExtension.java

```diff
@@ -88,7 +89,8 @@
  *       "TransactionRun" and "CommittedTransactionWithProof".
  * </ul>
  */
-public record LedgerExtension(List<RawLedgerTransaction> transactions, LedgerProof proof) {
+public record LedgerExtension(List<RawLedgerTransaction> transactions, LedgerProof proof)
```

Comment:
it's a bit weird that we're using this record as an event on its own :thinking: 
what do you think about adding a wrapper for it with a better name? e.g. `record VerifiedLedgerSyncResponse(ledgerExtension)`.
Possibly in a separate pr

Reply:
I think it is worth to add to separate ticket.